R/set-record-option.R

```diff
@@ -0,0 +1,110 @@
+
+#' Add or modify a record option
+#'
+#' Add a new record option or adjust the value/presence of an existing one.
+#'
+#' Note that only estimation, data, and table records are currently supported.
+#'
+#' Notes on `value`:
+#'  - We infer whether to use `option_value` or `option_flag` for *new* options based on the *class* of `value`:
+#'     - If you are trying to add an option with an assigned value (i.e. `MAXEVAL = 10`), `value` must be a character string or numeric value
+#'     - If you are trying to add a flag option (i.e. `NOABORT`), `value` should be logical (`T`/`F`).
+#'  - If `value` is set to `NULL`, the option will be removed (if found). `FALSE` is also acceptable for removing an existing option of class `nmrec_option_flag`.
+#'
+#' @param record An [nmrec_record] object.
```

Comment:
Let's spell this as `[option]` or `[nmrec_option]` for consistency with all the other doc instances.

Reply:
If you're referring to `nmrec_option_flag`, this has been deleted per your suggestion, but good to know going forward

Reply:
Sorry, I marked the wrong spot.  I was referring to `nmrec::option` a few lines below:

https://github.com/metrumresearchgroup/nmrec/blob/9e63bf0c05d30ea26764cf2d043880bfdc3d1616/R/set-record-option.R#L18